src/lib.rs

```diff
@@ -110,6 +77,57 @@ extern crate serde_derive;
 #[macro_use]
 mod macros;
 
+// ############################################################################
+// Public API
+// ############################################################################
+pub use self::{
+    chain::quorum_count,
```

Comment:
quorum_count only used in a mock test, does not seem to be used in vaults (first API use).